test/utils/actions.ts

```diff
@@ -161,9 +167,57 @@ export const App = {
       this.wait
     );
   },
+  openServiceWorkerTab: async function (this: mocha.Context) {
+    await this.driver.switchTo().newWindow('extension');
```

Comment:
This method accepts only `tab` or `window` value.

https://www.selenium.dev/documentation/webdriver/browser/windows/#create-new-window-or-new-tab-and-switch